# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xz/xz-noclobber-lzma.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur xz-4.999.9beta.orig//src/lzmainfo/Makefile.am xz-4.999.9beta//src/lzmainfo/Makefile.am
--- xz-4.999.9beta.orig//src/lzmainfo/Makefile.am	2009-08-27 13:07:12.000000000 -0230
+++ xz-4.999.9beta//src/lzmainfo/Makefile.am	2010-08-21 15:51:09.000000000 -0230
@@ -5,25 +5,25 @@
 ## You can do whatever you want with this file.
 ##
 
-bin_PROGRAMS = lzmainfo
+# bin_PROGRAMS = lzmainfo
 
-lzmainfo_SOURCES = lzmainfo.c
+# lzmainfo_SOURCES = lzmainfo.c
 
-lzmainfo_CPPFLAGS = \
+# lzmainfo_CPPFLAGS = \
 	-DLOCALEDIR=\"$(localedir)\" \
 	-I$(top_srcdir)/src/common \
 	-I$(top_srcdir)/src/liblzma/api \
 	-I$(top_builddir)/lib \
 	$(DYNAMIC_CPPFLAGS)
 
-lzmainfo_LDFLAGS = $(DYNAMIC_LDFLAGS)
-lzmainfo_LDADD = $(top_builddir)/src/liblzma/liblzma.la
+# lzmainfo_LDFLAGS = $(DYNAMIC_LDFLAGS)
+# lzmainfo_LDADD = $(top_builddir)/src/liblzma/liblzma.la
 
-if COND_GNULIB
-lzmainfo_LDADD += $(top_builddir)/lib/libgnu.a
-endif
+# if COND_GNULIB
+# lzmainfo_LDADD += $(top_builddir)/lib/libgnu.a
+# endif
 
-lzmainfo_LDADD += $(LTLIBINTL)
+# lzmainfo_LDADD += $(LTLIBINTL)
 
 
-dist_man_MANS = lzmainfo.1
+# dist_man_MANS = lzmainfo.1
diff -Nur xz-4.999.9beta.orig//src/scripts/Makefile.am xz-4.999.9beta//src/scripts/Makefile.am
--- xz-4.999.9beta.orig//src/scripts/Makefile.am	2009-08-27 13:07:12.000000000 -0230
+++ xz-4.999.9beta//src/scripts/Makefile.am	2010-08-21 15:46:18.000000000 -0230
@@ -9,46 +9,8 @@
 dist_man_MANS = xzdiff.1 xzgrep.1 xzmore.1 xzless.1
 
 install-exec-hook:
-	cd $(DESTDIR)$(bindir) && \
-	target=`echo xzdiff | sed '$(transform)'` && \
-	for name in xzcmp lzdiff lzcmp; do \
-		link=`echo $$name | sed '$(transform)'` && \
-		rm -f $$link && \
-		$(LN_S) $$target $$link; \
-	done && \
-	target=`echo xzgrep | sed '$(transform)'` && \
-	for name in xzegrep xzfgrep lzgrep lzegrep lzfgrep; do \
-		link=`echo $$name | sed '$(transform)'` && \
-		rm -f $$link && \
-		$(LN_S) $$target $$link; \
-	done && \
-	for name in xzmore xzless; do \
-		target=`echo $$name | sed '$(transform)'` && \
-		link=`echo $$name | sed 's/xz/lz/;$(transform)'` && \
-		rm -f $$link && \
-		$(LN_S) $$target $$link; \
-	done
 
 install-data-hook:
-	cd $(DESTDIR)$(mandir)/man1 && \
-	target=`echo xzdiff | sed '$(transform)'` && \
-	for name in xzcmp lzdiff lzcmp; do \
-		link=`echo $$name | sed '$(transform)'` && \
-		rm -f $$link.1 && \
-		$(LN_S) $$target.1 $$link.1; \
-	done && \
-	target=`echo xzgrep | sed '$(transform)'` && \
-	for name in xzegrep xzfgrep lzgrep lzegrep lzfgrep; do \
-		link=`echo $$name | sed '$(transform)'` && \
-		rm -f $$link.1 && \
-		$(LN_S) $$target.1 $$link.1; \
-	done && \
-	for name in xzmore xzless; do \
-		target=`echo $$name | sed '$(transform)'` && \
-		link=`echo $$name | sed 's/xz/lz/;$(transform)'` && \
-		rm -f $$link.1 && \
-		$(LN_S) $$target.1 $$link.1; \
-	done
 
 uninstall-hook:
 	cd $(DESTDIR)$(bindir) && \
diff -Nur xz-4.999.9beta.orig//src/xz/Makefile.am xz-4.999.9beta//src/xz/Makefile.am
--- xz-4.999.9beta.orig//src/xz/Makefile.am	2009-08-27 13:07:12.000000000 -0230
+++ xz-4.999.9beta//src/xz/Makefile.am	2010-08-21 15:47:40.000000000 -0230
@@ -66,22 +66,8 @@
 xzlinks = unxz xzcat lzma unlzma lzcat
 
 install-exec-hook:
-	cd $(DESTDIR)$(bindir) && \
-	target=`echo xz | sed '$(transform)'`$(EXEEXT) && \
-	for name in $(xzlinks); do \
-		link=`echo $$name | sed '$(transform)'` && \
-		rm -f $$link && \
-		$(LN_S) $$target $$link; \
-	done
 
 install-data-hook:
-	cd $(DESTDIR)$(mandir)/man1 && \
-	target=`echo xz | sed '$(transform)'` && \
-	for name in $(xzlinks); do \
-		link=`echo $$name | sed '$(transform)'` && \
-		rm -f $$link.1 && \
-		$(LN_S) $$target.1 $$link.1; \
-	done
 
 uninstall-hook:
 	cd $(DESTDIR)$(bindir) && \
diff -Nur xz-4.999.9beta.orig//src/xzdec/Makefile.am xz-4.999.9beta//src/xzdec/Makefile.am
--- xz-4.999.9beta.orig//src/xzdec/Makefile.am	2009-08-27 13:07:12.000000000 -0230
+++ xz-4.999.9beta//src/xzdec/Makefile.am	2010-08-21 15:48:58.000000000 -0230
@@ -12,7 +12,7 @@
 		$(xzdec_CPPFLAGS) $(CPPFLAGS) $(RCFLAGS) -i $< -o $@
 
 
-bin_PROGRAMS = xzdec lzmadec
+bin_PROGRAMS = xzdec
 
 xzdec_SOURCES = xzdec.c
 
@@ -49,11 +49,6 @@
 dist_man_MANS = xzdec.1
 
 install-data-hook:
-	cd $(DESTDIR)$(mandir)/man1 && \
-	target=`echo xzdec | sed '$(transform)'` && \
-	link=`echo lzmadec | sed '$(transform)'` && \
-	rm -f $$link.1 && \
-	$(LN_S) $$target.1 $$link.1
 
 uninstall-hook:
 	cd $(DESTDIR)$(mandir)/man1 && \
